.eslintignore

```diff
@@ -0,0 +1 @@
+packages/
```

Comment:
but why? :D if there is no lint script specified the linter does not run in the first place...

Reply:
It's super weird i've got an error that tonemate-utils doesn't specify an eslint config, though it didn't even specify any `lint` command in the first place :D. Maybe we should just add eslintconfigs there as well? I think the shared `server` eslintconfig is a good fit no?

Reply:
Here a picture of what happens if you change `tonemato-utils` and try to commit.
![output_lint_packages](https://user-images.githubusercontent.com/18482238/233000105-01506b31-d043-4acc-ba3e-de8e54540eb1.png)

Reply:
weird... never happened to me

Reply:
ahhhh I know why... thats my bad I configured lint-staged to check that package but never set up eslint there :D gonna add a fix for that in a separate pr